Bottom: 6eb0ffe1f9ccd21bca0eb1d0496b94a915924f08
Top:    a2c5d9a0bd1e9f6db6743a93abcef21264d5e1be
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2017-01-18 23:51:11 +0000

Refresh of toy-rendering-attempt

---

diff --git a/src/Makefile.am b/src/Makefile.am
index 8c9e089..6e3d281 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -24,12 +24,10 @@ bin_PROGRAMS=	pcb
 
 if DEBUG_BUILD
 # don't disable assert()
-AM_CFLAGS= -Wno-unused-parameter
 else
-AM_CFLAGS= -DNDEBUG -Wno-unused-parameter
+AM_CFLAGS= -DNDEBUG
 endif
 
-
 PCB_SRCS = \
 	action.c \
 	action.h \
